### DISCUSSION TITLE
alsa-sys and libudev-sys not found

### DISCUSSION THREAD

**Discussion by cybersoulK** [rust-cross/cargo-zigbuild#182] [Q&A]
this toolchain is great, but how about compiling audio (rodio/kira) and compiling controller input (gilrs)
these need the alsa-sys and libudev-sys.
I spent several days compiling them statically on an ubuntu machine, and after finishing compiling/linking, they segfault on my users.
I wonder what the solution would be to have the current toolchain compile for these crates.

**Comment by messense** [comment]
Sorry I don't know.
